src/common/schemas/Run.yml

```diff
@@ -23,14 +23,7 @@
       type: array
       readOnly: true
       items:
-        type: object
-        properties:
-          runID:
-            type: string
-          time:
-            type: string
-          message:
-            type: string
+        $ref: '#/components/schemas/LogEvent'
```

Comment:
This looks like the change that's failing CI, another `make generate` should fix it.

Reply:
Figured it out - should pass now